cypress/integration/destination/create_destination_spec.js

```diff
@@ -5,7 +5,8 @@ describe('Create Destination', () => {
   });
 
   it('renders the page and takes a screenshot', () => {
-    cy.getByTestId('TypePicker').should('contain', 'Email');
+    cy.getByTestId('CreateSourceDialog').should('contain', 'Email');
+    cy.wait(1000);
```

Comment:
If you don't wait, it snaps before the animation completes?

Reply:
> If you don't wait, it snaps before the animation completes?

Exactly :confused:

Reply:
https://percy.io/Redash/redash/builds/1539798

Reply:
The best way to tackle this I think, is to disable animations when testing.
Is there a way to insert sth like this for tests?
```css
* {
    animation-duration: 0s !important;
}
```

Reply:
Or perhaps have a less file that overrides:
```less
@animation-duration-slow: 0;
```

Reply:
I was passing through their docs today and I saw [this](https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Interacting-With-Elements):

"For example, when writing a `.click()` command, Cypress ensures that the element is able to be interacted with (like a real user would). It will automatically wait until the element reaches an “actionable” state by:
- Not being hidden
- Not being covered
- Not being disabled
- Not animating"

If this one is really due to the animation, it's an issue with Percy snapshot command, not with Cypress, so perhaps workaround this with `.click()` on, let's say, the Search input will be much easier than remove animations. WDYT?

Reply:
Interesting workaround. Try it and see if it works?

Btw, Cypress screenshots address this problem directly:
```js
Cypress.Screenshot.defaults({
  disableTimersAndAnimations: false
})
```